examples/java/.gitignore

```diff
@@ -0,0 +1,4 @@
+target/
```

Comment:
As your example provides a "test-unit" code sample, could you rename the folder name from `java` to `test_java_junit`?